src/libraries/Microsoft.PowerFx.Interpreter/Functions/Extensions.cs

```diff
@@ -10,7 +10,11 @@ internal static class Extensions
         internal static bool IsValid(this DateTime dateTime, EvalVisitor runner)
         {
             var tzi = runner.GetService<TimeZoneInfo>() ?? TimeZoneInfo.Local;
```

Comment:
>runner.GetService<TimeZoneInfo>() ?? TimeZoneInfo.Local [](http://example.com/codeflow?start=22&length=55)

I think we can use runner.TimeZoneInfo now here.

Reply:
updated this.